### PR TITLE
Add knative-eventing-kafka component

### PR DESCRIPTION
**Pull request by nachtmaar**
**Description**

Changes proposed in this pull request:

- Follow up PR on https://github.com/kyma-incubator/compass/pull/905
- The code in the PR already creates the installation-overrides, now we can enable the component such that we have Kafka based eventing in Kyma

**How to Test**
```shell
$ kyma test run knative-eventing-kafka -w
```

**Related issue(s)**

### CODE REVIEW

File: chart/compass/charts/kyma-environment-broker/values.yaml [hunk 1]
```diff
@@ -48,8 +48,10 @@ additionalRuntimeComponents:
     namespace: "kyma-system"
   - name: "compass-runtime-agent"
     namespace: "compass-system"
+  - name: "knative-eventing-kafka"
+    namespace: "knative-eventing"
 
-kymaVersion: "1.10.0"
+kymaVersion: "master-99c5dca2"
```
Comment (anishj0shi) [inline chart/compass/charts/kyma-environment-broker/values.yaml]: I'm not sure we should propose this SHA change. :p

Comment (nachtmaar) [reply chart/compass/charts/kyma-environment-broker/values.yaml]: It is used to set the `APP_KYMA_VERSION` environment variable in the KEB deployment. We need to update it to have the resources for knative-eventing-kafka inside the installer image. We added the component files in this [PR](https://github.com/kyma-project/kyma/pull/7486). So we need a newer Kyma version than 1.10. We could also use 1.11 as soon as it is out.

Comment (mszostok) [inline chart/compass/charts/kyma-environment-broker/values.yaml]:
```suggestion
kymaVersion: "1.11.0-rc2"
```

I've tested that with rc2 and more or less it's working

**Comment by mszostok** [comment]
please also update this chart on Kyma repo. Our pipelines are executed only when Kyma repo is updated: https://github.com/kyma-project/kyma/tree/master/resources/compass/charts/kyma-environment-broker

**Pull request merged**